MSAL/src/public/configuration/publicClientApplication/cache/MSALCacheConfig.h

```diff
@@ -59,6 +59,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, nullable) MSALSerializedADALCacheProvider *serializedADALCache;
 
+/*!
+ Array of SecTrustedApplicationsRef that is allowed to access the keychain elements
+ created by the keychain cache.
+ */
+@property (readonly, nonnull) NSArray *trustedApplications;
```

Comment:
Do we want to provide developers with any better helpers to create those? e.g. create from app path etc. The code is fairly uncommon, and it will be difficult for each developer to implement it.

Reply:
Added a link for apple documentation on SecTrustedApplicationCreateFromPath. I think that would be sufficient.

Reply:
Added a helper method that creates a list of trusted app instances (SecTrustedApplicationsRef) based on the apps at the given path in the file system.